### PR TITLE
Fix some logging issues and yum checks

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This pull request fixes:
- add some log issues
- add support for yum checks

### CODE REVIEW

**Comment by jscotka** [comment]
These changes seems well, it is better to move ENV handling to common module to have it on one place

File: moduleframework/common.py [hunk 1]
```diff
@@ -719,3 +718,24 @@ def get_module_type_base():
...
     return parent
+
+
+def get_docker_file(dir_name="../"):
+    """
+    Function returns full path to dockerfile.
+    :param dir_name: dir_name, where should be Dockerfile located
+    :return: full_path to Dockerfile
+    """
+    fromenv = os.environ.get("DOCKERFILE")
```
Comment (TomasTomecek) [inline moduleframework/common.py]: this should be documented

File: moduleframework/common.py [hunk 2]
```diff
@@ -719,3 +718,24 @@ def get_module_type_base():
...
+        dockerfile = fromenv
+        dir_name = os.getcwd()
+    else:
+        dir_name = os.path.abspath(dir_name)
+        dockerfile = DOCKERFILE
+    dockerfile = os.path.join(dir_name, dockerfile)
+
+    if not os.path.exists(dockerfile):
+        dockerfile = None
+        print_debug("Dockerfile should exists in the %s directory." % dir_name)
```
Comment (TomasTomecek) [inline moduleframework/common.py]: this should print absolute path to the directory, relative path will be confusing to users

File: moduleframework/common.py (hunk 1 above)
Comment (TomasTomecek) [reply moduleframework/common.py]: http://meta-test-family.readthedocs.io/en/latest/user_guide/environment_variables.html

Comment (phracek) [reply moduleframework/common.py]: Documented.

**Comment by TomasTomecek** [comment]
let's merge

**Pull request merged**